### DISCUSSION TITLE
Use CLI without login

### DISCUSSION THREAD

**Discussion by Spaceface16518** [supabase/supabase#7724] [Questions]
I want to run Jest unit tests in my Next.js + Supabase app. Can I configure GitH ub Actions CI to install supabase CLI, run services, and apply local migrations _without_ `supabase login` or `supabase db remote set`? Just `brew install supabase/tap/supabase` and `supabase start`?

**Comment by egor-romanov** [comment]
We have supabase-cli GitHub Action now, wdyt about trying it out?

Action: https://github.com/supabase/setup-cli

Examples: https://github.com/supabase/supabase-action-example/tree/main/.github/workflows
